config.json

```diff
@@ -9,24 +9,24 @@
     "message": "Hello! :wave: The Green Flag holder today is:\n`CDC:` <<2>>\n`Batch:` <<3>>\n\nUpdate rota <https://docs.google.com/spreadsheets/d/1XHDVi_VrUW0MX9emmpdqTsa78Xc6Qqpal24xidBXRb8/edit#gid=0|here>.\n\nSee details about what the Green Flag means <https://docs.google.com/document/d/1GOGUruD6ijSHyXgZLyJSRMyuSf0NVY1ICzl9PxaU208/edit|here>."
 },
 {
-    "name": "Integrations",
-    "description": "The integrations rota, posts to #int-incidents-flex-squad",
+    "name": "Modular Connectors",
+    "description": "The Modular Connectors rota, posts to #int-conjure-squad",
```

Comment:
I don't think I can see this channel, is it a private one @nixondante ?

Reply:
Ahh yeah it's a private one:
<img width="363" alt="image" src="https://github.com/johnathanlaw/rotabot/assets/155438985/159c9261-88da-4062-b323-374ab9d0ec5e">

Reply:
Just realised you needed to be added to the squad - done that now 😄

Reply:
Yeah I need to be in it unfortunately! - Thanks for adding, thats it muted now :) Will squash and merge!